tests/pyrebasetrial.py

```diff
@@ -0,0 +1,67 @@
+"""
+Standalone program to test the usage of pyrebase
+Pyrebase package helps python interact with Google firebase realtime database
+Features:
+ - It can upload files to the the storage bucket
+ - It can authenticate users with thier email id
+ - It can push, pull, and modify data in the realtime database
+ - It can create a listener in the real time database and run code when a change is detected
+For this project, modification of realtime database and creation of listeners is nessesary
+
+As this is the backend code, authentication for this program to access the firebase database should not be nessesary
+This is why a service account should be generated for this program
+requirements: pyrebase4 (https://github.com/nhorvath/Pyrebase4)
+"""
+import time
+
+# Importing pyrebase
+import pyrebase
+
+# Getting creds from the creds file
+with open("creds.txt") as file:
```

Comment:
for credentials/config stuff try using `yaml` or `toml` files, more human readable and makes code a lil cleaner

Reply:
I'll change it in the next commit